docs/reST/ref/draw.rst

```diff
@@ -64,12 +63,9 @@ object around the draw calls (see :func:`pygame.Surface.lock` and
          | if ``width > 0``, used for line thickness
          | if ``width < 0``, nothing will be drawn
          |
+      
+      .. versionchanged:: 2.1.1 Drawing rects with width now draws the width correctly inside the rect's area, rather than using an internal call to draw.lines(), which had half the width spill outside the rect area.
```

Comment:
That's what I was trying to talk to you about on discord! I don't know how / if there is a way to put one of these tags on multiple lines!

Reply:
```suggestion
      .. versionchanged:: 2.1.1 
          Drawing rects with width now draws the width correctly inside the 
          rect's area, rather than using an internal call to draw.lines(), 
          which had half the width spill outside the rect area.
```
Should do?

Reply:
Ugh, idk why I was certain that wouldn't work.

I bet sphinx 1.8 or something didn't work with it, I swear I've tried and failed with this before.